src/main/java/com/airlines/model/Position.java

```diff
@@ -0,0 +1,10 @@
+package com.airlines.model;
+
+public enum Position {
+    Captain,
+    FO,
```

Comment:
Try to use more declarative and human reading values, if it will be in code:

if (position != SO && position != FE) {
it would be very hard to understand what is going on

Reply:
done in https://github.com/pielena/airlines-jdbc-app/commit/bc4e257c81648513d3a0f4bc5de1c6e21696c76a